### PR TITLE
ft: add delete store functionality

### PR DESCRIPTION
**Pull request by aapetsi**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aapetsi** [comment]
[/profile](https://trello.com/c/UVsiHdjA/74-profile)

**Review by shaunorpen — CHANGES_REQUESTED** [review]
Bit confused about this PR. The title is delete store. The branch is delete account. Which is it?

Some questions and suggestions:

1. There's no 'Are you sure' message when you click 'delete account'. You just get a pop-up saying your account has been deleted. I think we should have a confirmation dialog.

2. After the account is deleted I'm redirected to dashboard. I would have thought I should be redirected to login / register? I think deleting account should clear the token from local storage otherwise when you visit / you get redirected to dashboard, which looks broken as there's no longer any account data to display.

**Comment by DimejiAre** [comment]
@aapetsi A modal was created for the delete item functionality. The one you used for the delete account looks very different. Try and reuse already existing components/design for uniformity and consistency across the app. It also saves time.

**Comment by aapetsi** [comment]
> @aapetsi A modal was created for the delete item functionality. The one you used for the delete account looks very different. Try and reuse already existing components/design for uniformity and consistency across the app. It also saves time.

@DimejiAre it's been fixed

File: src/components/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -47,6 +55,23 @@ const EditProfile = props => {
     history.push('/')
   }
 
+  const handleDeleteAccount = () => {
+    const { confirm } = Modal
+    confirm({
+      title: 'Are you sure you want to delete your account item?',
```
Comment (shaunorpen) [inline src/components/EditProfile/EditProfile.js]: Suggestion: Update this to read 'Are you sure you want to delete your account?'.

**Review by shaunorpen — CHANGES_REQUESTED** [review]
Just a minor update to the delete modal to suggest. Great work!

**Review by shaunorpen — APPROVED** [review]

**Pull request merged**